specs/tables.md

```diff
@@ -96,6 +101,10 @@ Type sizes:
 | $counter | $isWrite    | Stack                      | $callID  |          | $stackPointer              |             | $value     | 0          | 0        | 0               |
 | $counter | $isWrite    | Memory                     | $callID  |          | $memoryAddress             |             | $value     | 0          | 0        | 0               |
 | $counter | $isWrite    | AccountStorage             |          | $address |                            | $storageKey | $value     | $valuePrev | $txID    | $CommittedValue |
+|          |             |                            |          |          |                            |             |            |            |          |                 |
+| $counter | true        | TxLog                     |$log_index | 0        |  $address                  |             | $value     | 0          | 0        | 0               |
+| $counter | true        | TxLog                     |$log_index | $index   |  $Topics                   |             | $value     | 0          | 0        | 0               |
+| $counter | true        | TxLog                     |$log_index | $index   |  $Data                     |             | $value     | 0          | 0        | 0               |
```

Comment:
```suggestion
| $counter | true        | TxLog                     |$logID | 0        |  Address                  |             | $value     | 0          | 0        | 0               |
| $counter | true        | TxLog                     |$logID | $topicIndex   |  Topic                   |             | $value     | 0          | 0        | 0               |
| $counter | true        | TxLog                     |$logID | $byteIndex   |  Data                     |             | $value     | 0          | 0        | 0               |
```

Reply:
and also need to add an additional row for `TopicSize` indicating the number of topics for this log

Reply:
updated to use suggestion change for renaming.   For  `TopicSize` at the beginning I add it , but I don't see its usage in circuit as circuit can handle dynamic topic count, so I remove it later,  let me know if you still think it is needed !

Reply:
As per offline discussion, I agree that `TopicSize` is not necessary at this moment.